### PR TITLE
Disable the clock_gettime syscall on amd64 and use walltime1() instead

### PR DESCRIPTION
**Pull request by samhocevar**

### CODE REVIEW

**Comment by hajimehoshi** [comment]
This is tricky. This assumes that PlayStation is amd64 while Switch is arm64, right?

**Comment by hajimehoshi** [comment]
> This is tricky. This assumes that PlayStation is amd64 while Switch is arm64, right?

Aha, I understand this PR does not affect arm64, as this PR just changes `linux && amd64` files.

File: 1.19_linux/runtime/time_linux_amd64.s.patch [hunk 1]
```diff
@@ -0,0 +1,4 @@
+//--from
+//go:build !faketime
+//--to
+//go:build disable
```
Comment (hajimehoshi) [inline 1.19_linux/runtime/time_linux_amd64.s.patch]: Use `ignore`

File: 1.19_linux/runtime/timeasm.go.patch [hunk 2]
```diff
@@ -0,0 +1,4 @@
+//--from
+//go:build !faketime && (windows || (linux && amd64))
+//--to
+//go:build disable
```
Comment (hajimehoshi) [inline 1.19_linux/runtime/timeasm.go.patch]: `//go:build !faketime && windows`?

Comment (samhocevar) [reply 1.19_linux/runtime/timeasm.go.patch]: Fixed!

File: 1.19_linux/runtime/time_linux_amd64.s.patch (hunk 1 above)
Comment (samhocevar) [reply 1.19_linux/runtime/time_linux_amd64.s.patch]: Fixed!

**Review by hajimehoshi — APPROVED** [review]
LGTM, thanks!

**Pull request merged**